lib/Config/Hooks.php

```diff
@@ -168,11 +168,16 @@ function setScreenOption($status, $option, $value) {
 
   function setupPostNotifications() {
     foreach(WPPosts::getTypes() as $post_type) {
-      add_filter(
+      add_action(
```

Comment:
Do we really need to keep this hooks?
[Here ](https://developer.wordpress.org/reference/functions/wp_transition_post_status/#source) you can see that both actions are triggered together. So this  hooking to`'publish_post`seems to be redundant here.
It also says that this transition is triggered always even when status stays the same:
```
When a post is saved, the post status is "transitioned" from one status to another, though this does not always mean the status has actually changed before and after the save. 
```